server/app/models/grid_domain_authorization.rb

```diff
@@ -13,6 +13,7 @@ class GridDomainAuthorization
   field :challenge, type: Hash
   field :challenge_opts, type: Hash # TODO encrypt?
   field :authorization_type, type: String, default: 'tls-sni-01'
+  field :expires_at, type: DateTime
```

Comment:
`Time` vs `DateTime` ?

Reply:
Right... must have been copypasta from the `Certificate.valid_until` model.